source/usage-examples/struct-tagging.txt

```diff
@@ -0,0 +1,67 @@
+==========================
+Set Custom Driver Behavior
+==========================
+
+.. default-domain:: mongodb
+
+You can customize the way that the Go Driver converts between Go
+structs and :manual:`BSON </reference/bson-types/>` using 
+**struct tags**, which are optional instructions you can use with struct
+fields.
```

Comment:
consider adding an additional sentence here to provide more context on the use cases for struct tagging. The examples make it pretty clear, but I think this would help users extrapolate this feature to their own use cases.